blazor/diagram/symbol-palette.md

```diff
@@ -1340,6 +1340,115 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ![Symbol with Description in Blazor Diagram](images/blazor-diagram-symbol-description.png)
 
+## Appearance of symbol description
+
+You can customize the appearance of a symbol description in the symbol palette by adjusting the following properties:
```

Comment:
Customize the appearance of a symbol description in the symbol palette by adjusting the following properties: